Sources/Util/IndexTree.swift

```diff
@@ -283,7 +283,7 @@ extension IndexTreeNode {
     /**
      * `insertAfter` inserts the given node after the given child.
      */
-    func insertAfter(newNode: Self, referenceNode: Self) throws {
+    func insertAfter(_ newNode: Self, _ referenceNode: Self) throws {
```

Comment:
How about
```swift
func insert(_ newNode: Self, after childNode: Self) throws
```

Reply:
As mentioned earlier.  I will apply the swift style to a stable version later.